docs/platforms/javascript/guides/sveltekit/manual-setup.mdx

```diff
@@ -467,47 +517,31 @@ export const load = wrapServerLoadWithSentry(({ event }) => {
 });
 ```
 
-#### Configure Client-side `fetch` Instrumentation
+#### Instrument Server Routes
 
 <Note>
 
-Available since version `7.91.0`
+Available since `8.24.0`
 
 </Note>
 
-The `sentryHandle` function you added to your `handle` hook in `hooks.server.ts` during [server-side setup](#server-side-setup) injects a small inline `<script>` tag into the HTML response of the server.
-This script attempts to proxy all client-side `fetch` calls, so that `fetch` calls inside your `load` functions are captured by the SDK.
-However, if you configured CSP rules to block inline fetch scripts by default, this script will be [blocked by the browser](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/script-src#unsafe_inline_script).
-To enable the script, you need to add an exception for the `sentryHandle` script:
-
-First, specify your nonce in the `fetchProxyScriptNonce` option in your `sentryHandle` call:
-
-```javascript {filename:hooks.server.ts}
-// Add the nonce to the <script> tag:
-export const handle = sentryHandle({ fetchProxyScriptNonce: "<your-nonce>" });
-```
+You can also manually instrument [server (API) routes ](https://kit.svelte.dev/docs/routing#server) with the SDK.
+This is useful if you have custom server routes that you want to trace or if you want to capture `error()` calls within your server routes
 
-Then, adjust your [SvelteKit CSP configuration](https://kit.svelte.dev/docs/configuration#csp):
+```javascript {filename:+server.js} {1,3}
+import { wrapServerRouteWithSentry } from "@sentry/sveltekit";
 
-```javascript {svelte.config.js} {5}
-const config = {
-  kit: {
-    csp: {
-      directives: {
-        "script-src": ["nonce-<your-nonce>"],
-      },
-    },
-  },
-};
+export const GET = wrapServerRouteWithSentry(async () => {
+  // your endpoint logic
+  return new Response("Hello World");
+});
 ```
 
-##### Disable Client-side `fetch` Proxy Script
+```typescript {filename:+server.ts} {1,3}
```

Comment:
Why is this duplicated for ts and js?

Reply:
because you can use JS and TS simultaneously in SvelteKit. But most of these snippets have no TS-specific syntax, so it might be better to switch to a shared `filename.(js|ts)` snippet

Reply:
I'm realizing, we're a bit inconsistent with the languages and even diverge a bit. I'll dedupe the snippets.

Reply:
Opened an additional PR because I streamlined some other things while removing the TS stuff: https://github.com/getsentry/sentry-docs/pull/11038